CardanoSharp.Blockfrost.Sdk/Contracts/StakepoolExtended.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.Json.Serialization;
+using System.Threading.Tasks;
+
+namespace CardanoSharp.Blockfrost.Sdk.Contracts
+{
+    public class StakepoolExtended
+    {
+        /// <summary>
+        /// Bech32 encoded pool ID
+        /// </summary>
+        [JsonPropertyName("pool_id")] public string PoolId { get; set; }
+
+        /// <summary>
+        /// Hexadecimal pool ID.
+        /// </summary>
+        [JsonPropertyName("hex")] public string Hex { get; set; }

```

Comment:
i dont see hex in the stake pool extended info in BF docs, but maybe im looking in the wrong place?

Reply:
You're right, it's not explicitly listed in the example values, but I generated the contracts based on the OpenAPI (swagger) specs and in there, it's listed as a property:

![image](https://user-images.githubusercontent.com/20597158/197072554-0b090beb-2918-4ed1-b02c-8c90769cfcd9.png)

I think they just omitted/forgot to add it in the sample values for that type, because it does indeed get returned by blockfrost as per the openapi contract:

![image](https://user-images.githubusercontent.com/20597158/197072968-d99b4c39-dee9-41ce-a0e8-e8aeb7bf10e6.png)

to test: https://cardano-mainnet.blockfrost.io/api/v0/pools/extended?count=100&page=4&order=asc

Reply:
ok awesome